utils/profiles.ts

```diff
@@ -30,7 +32,18 @@ export function apiProfileToProfile(
     isEmailValidated,
     isInboxEnabled: apiProfile.is_inbox_enabled,
     isWebhookEnabled: apiProfile.is_webhook_enabled,
-    preferredLanguages: apiProfile.preferred_languages
+    preferredLanguages: apiProfile.preferred_languages,
+    servicePreferencesSettings:
+      apiProfile.service_preferences_settings.mode ===
```

Comment:
What if `service_preferences_settings` is `undefined`?

Reply:
We keep the old one, be it the default or another one. Does it seem legit?